### PR TITLE
update open graph meta tags attributes from name to property

### PR DESCRIPTION
**Pull request by greybax**
Fix for issue #9288 
Replaced "name=og:<some-name>" -> "property=og<some-name>."

### CODE REVIEW

**Comment by pieh** [comment]
Hmm, is `name` not working?

**Comment by greybax** [comment]
If you are using `og` prefix they should be with `property` meta tag.
Ex:
`name="title"`
or
`property="og:title"`

File: www/src/templates/template-starter-page.js [hunk 1]
```diff
@@ -60,9 +60,10 @@ class StarterTemplate extends React.Component {
             <Helmet>
               <title>{`${repoName}: Gatsby Starter`}</title>
               <meta
-                name="og:image"
-                content={screenshotFile.childImageSharp.fluid.src}
+                property="og:image"
+                content={`../logo.svg`}
```
Comment (DSchau) [inline www/src/templates/template-starter-page.js]: Why this change?

File: www/src/templates/template-docs-markdown.js [hunk 2]
```diff
@@ -68,10 +68,12 @@ class DocsTemplate extends React.Component {
         <Helmet>
           <title>{page.frontmatter.title}</title>
           <meta name="description" content={page.excerpt} />
-          <meta name="og:description" content={page.excerpt} />
+          <meta property="og:description" content={page.excerpt} />
+          <meta property="og:title" content={page.frontmatter.title} />
+          <meta property="og:type" content="article" />
+          <meta property="og:image" content={`../../logo.svg`} />
```
Comment (DSchau) [inline www/src/templates/template-docs-markdown.js]: Same here.. ? I'm not 100% this will be picked up, either.

File: www/src/views/starter-library/index.js [hunk 3]
```diff
@@ -27,15 +27,17 @@ class StarterLibraryPage extends Component {
...
             content={`Gatsby Starters: ${filtersApplied}`}
           />
           <meta
             name="twitter:description"
             content={`Gatsby Starters: ${filtersApplied}`}
           />
-          <meta name="og:title" content={filtersApplied} />
-          <meta name="og:type" content="article" />
+          <meta property="og:title" content={filtersApplied} />
+          <meta property="og:image" content={`../../logo.svg`} />
```
Comment (DSchau) [inline www/src/views/starter-library/index.js]: Yeah, so I'm pretty sure this won't work. This isn't _require_'d so this is going to remain a relative path. This would presume that two levels above `https://www.gatsbyjs.org/starters` would contain a logo.svg, which isn't the case, correct?

File: www/src/templates/template-starter-page.js (hunk 1 above)
Comment (fk) [reply www/src/templates/template-starter-page.js]: We have https://www.gatsbyjs.org/monogram.svg in https://github.com/gatsbyjs/gatsby/tree/master/www/static if that helps? Also could put a `PNG` or `JPG` version of the monogram or full logo (including the "Gatsby" wordmark) in there?

Comment (fk) [reply www/src/templates/template-starter-page.js]: …but I agree with Dustin, we should pass on the screenshot of the starter here?!

File: www/src/templates/template-docs-markdown.js (hunk 2 above)
Comment (greybax) [reply www/src/templates/template-docs-markdown.js]: I supposed that this relative path will be resolved in accordance with project structure. Based on file where I've made this change..

File: www/src/templates/template-starter-page.js (hunk 1 above)
Comment (greybax) [reply www/src/templates/template-starter-page.js]: reverted back to `screenshotFile.childImageSharp.fluid.src`

File: www/src/views/starter-library/index.js (hunk 3 above)
Comment (greybax) [reply www/src/views/starter-library/index.js]: Not sure which one we need to use here..
